README.md

```diff
@@ -1,2 +1,108 @@
-# service-catalog-tester
-Contains reproducible test-cases for service-catalog.
+# Service Catalog Tester
+
+## Overview
+
+This project contains reproducible test-cases for service-catalog.
+
+## Prerequisites
+
+To set up the project, use these tools:
+* Version 1.9 or higher of [Go](https://golang.org/dl/)
+* The latest version of [Docker](https://www.docker.com/)
+* The latest version of [Dep](https://github.com/golang/dep)
+
+## Usage
+
+This section explains how to use the Etcd Backup tool.
```

Comment:
Maybe explain briefly what this tool is used for? What's the connection between Etcd Backup and the Service Catalog Tester?

Reply:
bug, there should be `Service Catalog Tester`, final sentence:
`This section explains how to use the Service Catalog Tester tool.`